jasa/paper-jasa.tex

```diff
@@ -47,7 +47,7 @@
 \if1\blind
 {
   \title{\bf Approximating Likelihood Ratios with Calibrated Discriminative Classifiers}
-  \author{Kyle Cranmer and Gilles Louppe\\
+  \author{Kyle Cranmer, Juan Pavez and Gilles Louppe\\

```

Comment:
Hi @glouppe great!, reading now. A quick question .. do you need any help in the section of toy experiments? ... also, with diagnosis you mean training a classifier on weighted data and looking at AUC values?